.expeditor/config.yml

```diff
@@ -1,57 +1,40 @@
 project:
-  alias: omnitruck
+  alias: omnitruck-k8s
```

Comment:
This alias should remain `omnitruck`

Reply:
Thanks , Its now updated

Reply:
Looks like there are some git conflicts that need to be resolved.